calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,169 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports radars whose detection interface includes distance and azimuth angle, but do not offer elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the [radar reflector](#radar-reflector) (calibration target).
```

Comment:
the ARS408 radar
or
ARS408 radars

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/11487a6f94821fed4a42cecb240111303f8ececc